pandas/core/dtypes/common.py

```diff
@@ -1219,9 +1219,9 @@ def is_numeric_dtype(arr_or_dtype) -> bool:
     )
 
 
-def is_any_numeric_dtype(arr_or_dtype) -> bool:
+def is_any_real_numeric_dtype(arr_or_dtype) -> bool:
```

Comment:
@phofl on the off chance someone is actually using this, maybe it's safer if we make an alias `is_any_real_numeric_dtype = is_any_numeric_dtype` the "public" version?

Reply:
It was only added a couple of days ago, it's especially in no release yet, so I think we are good.

Reply:
Ah okay gotcha. Fine to change then